conf/server/server_full.conf

```diff
@@ -803,6 +803,32 @@ plugins {
     #         namespace = "sandbox"
     #     }
     # }
+
+    # BundlePublisher "aws_s3": A bundle publisher that puys the latest trust bundle
+    # contents into an object in Google Cloud Storage.
+    # BundlePublisher "aws_s3" {
+    #     plugin_data {
+    #         # region: AWS region to store the trust bundle. Default: "".
+    #         # region = "us-east-1"
+
+    #         # access_key_id: AWS access key id. Default: value of
+    #         # AWS_ACCESS_KEY_ID environment variable.
+    #         # access_key_id = ""
+
+    #         # secret_access_key: AWS secret access key. Default: value of
+    #         # AWS_SECRET_ACCESS_KEY environment variable.
+    #         # secret_access_key = ""
+
+    #         # bucket: The Amazon S3 bucket name to which the trust bundle is uploaded. Default: "".
+    #         # bucket = "spire-bundle-1"
+
+    #         # object_key: The object key inside the bucket. Default: "".
+    #         # object_key = "example.org"
+
+    #         # format: Format in which the trust bundle is stored, <spiffe | jwks | pem>. Default: "".
```

Comment:
What format is defaulted to when unset?

Reply:
The plugin requires that a format is explicitly configured, so there is not a default format set.

Reply:
Do you think a default would make sense here, `pem` for instance since it's the most well understood format?

Reply:
If we have a default, I would probably suggest to be the `spiffe` format. PEM does not allow the encoding of JWT authorities, which may be needed. Also, agents can be configured to use `spiffe` format for the initial SPIRE Server trust bundle.
The idea behind requiring a value here is to avoid the inadvertently misconfiguration of the plugin, but we can revisit that decision.

Reply:
> The idea behind requiring a value here is to avoid the inadvertently misconfiguration of the plugin

No, that's ok. I think this convinced me to be honest.